Packs/SalesforceIndicators/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Salesforce Indicators",
+    "description": "This pack contains 1 integration, 1 indicator classifier and 1 indicator mapper.\n\nThe Salesforce Indicator Integration allows you to pull in Salesforce objects as indicators. The classifier and mapper are used to classify the indicator and map the relevant fields.",
```

Comment:
The PR contains only the integration, is any more files should be attached here?

Reply:
I'll change the description. I originally created a blank classifier and blank mapper, which I know realise was pointless 😄